### DISCUSSION TITLE
P2 detection output

### DISCUSSION THREAD

**Discussion by Argentum471** [zhiqwang/yolov5-rt-stack#402] [General]
Hello @zhiqwang,
Im interested to use P2 layer for detecting very small objects in my project. Is it possible to add this in yolort?

**Comment by zhiqwang** [comment]
Hi @Argentum471 ,

`yolort` does not currently support custom P2 structures, but it would not be very difficult to do so. Did you get your P2 layer from the official YOLOv5's [hub](https://github.com/ultralytics/yolov5/blob/master/models/hub/yolov5-p2.yaml)? 

I think it will be better to discuss this ticket in the issue to make it easier to track it, would you like to resubmit a new ticket labeled enhancement [at here](https://github.com/zhiqwang/yolov5-rt-stack/issues/new?assignees=&labels=&template=feature-request.yml), and it would be better to mention your subsequent usage as well. Because the changes we make for different backends compatibility for the `LibTorch`, `ONNX Runtime` and `TensorRT` will require more work, but it would be much easier to support just one of them.
